### ISSUE TITLE
[a11y] Reserved checkmark in Manage Packages page is not accessible

### ISSUE THREAD

**Issue by drewgillies** [NuGet/NuGetGallery#8745]
This checkmark: 
![image](https://user-images.githubusercontent.com/14225979/129675823-0249e0df-0881-43b1-af49-9ccf164252f4.png)

...needs the same treatment as the checkmarks here: https://github.com/NuGet/NuGetGallery/pull/8521 and here: https://github.com/NuGet/NuGetGallery/pull/8622
